Solutions/Palo Alto Cortex XDR CCP/Package/mainTemplate.json

```diff
@@ -2396,7 +2396,7 @@
             },
             "apiToken": {
               "defaultValue": "apiToken",
-              "type": "string",
+              "type": "securestring",
```

Comment:
any string here should be "secureString". we dont want to log any value (to avoid any secure issue)

Reply:
@sagamzu, updated to "secureString" to "securestring".